frontend-implicaction/src/app/admin/companies/components/companies-table/companies-table.component.ts

```diff
@@ -66,7 +67,7 @@ export class CompaniesTableComponent extends BaseWithPaginationComponent<Company
           })
       );
 
-    this.getFilterFromQueryParams().then(() => this.filterService.criteria = this.criteria);
+    this.getFilterFromQueryParams(this.filterKeys).then(() => this.filterService.criteria = this.criteria);
```

Comment:
```suggestion
    this.getFilterFromQueryParams(['keyword']).then(() => this.filterService.criteria = this.criteria);
```
Je me demande si c'est pas plus simple de passer directement comme ça, sans utiliser une variable ?